web/src/main/java/org/cbioportal/web/parameter/MultipleStudiesEnrichmentFilter.java

```diff
@@ -0,0 +1,32 @@
+package org.cbioportal.web.parameter;
+
+import org.cbioportal.model.Entity;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.List;
+
+public class MultipleStudiesEnrichmentFilter {
+    
+    @NotNull
+    @Size(min = 1)
+    private List<Entity> set1;
```

Comment:
Same concerns with variable naming as above

Reply:
it's definitely hard to read with `Entity` and `set` both vague